Makefile.global.in

```diff
@@ -17,6 +17,12 @@ postgres_abs_builddir:=@POSTGRES_BUILDDIR@
 PG_CONFIG:=@PG_CONFIG@
 PGXS:=$(shell $(PG_CONFIG) --pgxs)
 
+ifeq ($(enable_coverage),yes)
+	export PG_CPPFLAGS += -ftest-coverage -fprofile-arcs
```

Comment:
Figured out this was coming from PostgreSQL's `enable-coverage` flag. I'm OK copying it.